dev-db/mydumper/mydumper-0.10.7.2.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+MY_P="${PN}-$(ver_rs 3 -)"
+
+DESCRIPTION="A high-performance multi-threaded backup (and restore) toolset for MySQL"
+HOMEPAGE="https://github.com/maxbube/mydumper"
+SRC_URI="https://github.com/maxbube/${PN}/archive/v${MY_PV}.tar.gz -> ${MY_P}.tar.gz"
```

Comment:
`${MY_PV}`?
```
>>> Downloading 'https://github.com/maxbube/mydumper/archive/v.tar.gz'
2021-08-07 05:39:27 ERROR 404: Not Found.
```

Reply:
Seems like I had MY_PV in the ebuild before and removed just by accident and since the file was downloaded the building passed, oh, stupid mistake. MY_PV is needed as version is 0.10.7-2.

Also haven't noticed the PN plaintexting, thanks for that.